tests/support/Mock/TestController.php

```diff
@@ -16,7 +16,7 @@ public function testRequestWithAnnotations(TestRequestWithAnnotations $request)
     {
     }
 
-    public function testRequestWithContract(TestContract $contract)
+    public function testRequestWithContract(TestContract $contract, string $param)
```

Comment:
The new string native type covers:
https://github.com/RonasIT/laravel-swagger/blob/688fe0b4146b8f9a71abb95e7971c47dcbbc1655/src/Traits/GetDependenciesTrait.php#L36-L38